v3/src/components/graph/plots/bar-chart/bar-chart-model.ts

```diff
@@ -16,31 +16,68 @@ export const BarChartModel = DotChartModel
   .named("BarChartModel")
   .props({
     type: typesPlotType("barChart"),
-    breakdownType: types.maybe(types.enumeration([...BreakdownTypes])),
+    breakdownType: types.optional(types.enumeration([...BreakdownTypes]), "count"),
     expression: types.maybe(Formula)
   })
+  .actions(self => ({
+    setBreakdownType(type: BreakdownType) {
+      self.breakdownType = type
+    },
+    setExpression(expression: IFormula) {
+      self.expression = expression
```

Comment:
Should setting the expression also set the `breakdownType` to `formula`? If matching v2 behavior were the goal, the answer would be no. Because `breakdownType` was in the base class, switching from `BarChart` to `ComputedBarChart` wouldn't clear the `breakdownType`, but going in reverse _would_ clear the expression.

Reply:
I think sticking with V2 behavior is fine.